paper.tex

```diff
@@ -83,125 +83,129 @@ \subsection{Terminology}
 
 \section{Motivation}
 
+
+\begin{quotation}
+      \textit{Better Software, Better Research}\\(Mission statement of the UK Software Sustainability Institute)
```

Comment:
This slogan is under CC-BY. I once asked and they replied that I should use it with "copyrighted by Carole Goble and the Software Sustainability Institute under the CC-BY license: https://www.software.ac.uk/resources/publications/better-software-better-research". Should we make this a footnote then, instead of the the SSI mention in brackets?